smriprep/cli/run.py

```diff
@@ -397,9 +396,8 @@ def build_workflow(opts, retval):
         logger.log(25, 'Running --reports-only on participants %s', ', '.join(subject_list))
         if opts.run_uuid is not None:
             run_uuid = opts.run_uuid
-        retval['return_code'] = generate_reports(
-            subject_list, str(output_dir), str(work_dir), run_uuid,
-            config=pkgrf('smriprep', 'data/reports/config.json'))
```

Comment:
This was still using the old system. I'm afraid the --reports-only option is not sufficiently tested.